### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by rohanali1798**
Update README for Alien4Cloud user instructions. 
@kinow

### CODE REVIEW

File: Pillar_II/esm/README.md [hunk 1]
```diff
@@ -348,27 +348,32 @@ may be useful for others with issues:
 
 #### Running from Alien4Cloud
 
-There is an environment initialization script:
-`/gpfs/projects/dese28/eflows4hpc/esm/fesom2/env.sh`
+Firstly, you must have a Alien4Cloud user account and have a vault ID (SSH key) and you have to add it to the `authorized_keys` file on the systems you want to run your workflows.
```
Comment (kinow) [inline Pillar_II/esm/README.md]: s/a Alien/an Alien

Can you also break the long line into something around ~60-200? It's useful for Git diff and reading on terminal/some editors - https://sembr.org/

File: Pillar_II/esm/README.md [hunk 2]
```diff
@@ -348,27 +348,32 @@ may be useful for others with issues:
...
-        --pythonpath=/gpfs/projects/dese28/eflows4hpc/esm/fesom2/src:/apps/HECUBA/2.1_intel/compss \
-        --env_script=/gpfs/projects/dese28/eflows4hpc/esm/fesom2/env.sh \
-        --storage_props=/gpfs/projects/dese28/eflows4hpc/esm/fesom2/src/hecuba_lib/storage_props.cfg --storage_home=/apps/HECUBA/2.1_intel/compss
+docker run ghcr.io/eflows4hpc/hpcwaas-api:main-cli --api_url https://eflows4hpc.bsc.es/waas -u [your alien4cloud user]:***********= ssh_keys key-gen
 ```
+Running this will get you a SSH key ID which will be used in setting up TOSCA application. 
+
+##### Create your application 
+
+Login to your Alien4Cloud account and create a new application based on an existing topology template. Choose `fesom2 parameterised` as your topology template and proceed creating your application.
```
Comment (kinow) [inline Pillar_II/esm/README.md]: s/Login/Log in, and ditto the break lines :point_up:

File: Pillar_II/esm/README.md [hunk 3]
```diff
@@ -348,27 +348,32 @@ may be useful for others with issues:
...
+
+##### Configuring topology and running your application
+
+Once you create your application. You will see an option to edit your `Environment`.
```
Comment (kinow) [inline Pillar_II/esm/README.md]: s/. You/, you

File: Pillar_II/esm/README.md [hunk 4]
```diff
@@ -348,27 +348,32 @@ may be useful for others with issues:
...
+
+Login to your Alien4Cloud account and create a new application based on an existing topology template. Choose `fesom2 parameterised` as your topology template and proceed creating your application.
+
+##### Configuring topology and running your application
+
+Once you create your application. You will see an option to edit your `Environment`. 
+
+Proceed to edit your environment and navigate to `Prepare next deployment`. Here you will see options to modify your `Topology`, `Inputs`, `Location` etc.
+
+Feel free to edit your topology if needed. Since we created is based on `fesom2 parameterised` template so it has all the paths and parameters like --start_date --hpc etc pre-configured that you can see in the `arguments` field. The main script that is configured to be launched in this template is `/gpfs/projects/dese28/eflows4hpc/workflow-registry/Pillar_II/esm/src/launch_fesom2.sh` which you can see in the `command` field in your topology.
```
Comment (kinow) [inline Pillar_II/esm/README.md]: s/is based/it based ?

s/template so it has/template, it has/?

s/like --start_date --hpc etc/like `--start_date` `--hpc` etc.

And add some breaks to the line (other paragraphs can be used as example).

File: Pillar_II/esm/README.md [hunk 5]
```diff
@@ -348,27 +348,32 @@ may be useful for others with issues:
...
+
+Proceed to edit your environment and navigate to `Prepare next deployment`. Here you will see options to modify your `Topology`, `Inputs`, `Location` etc.
+
+Feel free to edit your topology if needed. Since we created is based on `fesom2 parameterised` template so it has all the paths and parameters like --start_date --hpc etc pre-configured that you can see in the `arguments` field. The main script that is configured to be launched in this template is `/gpfs/projects/dese28/eflows4hpc/workflow-registry/Pillar_II/esm/src/launch_fesom2.sh` which you can see in the `command` field in your topology. 
+
+Save your changes to the topology if any and proceed to `Inputs` and fill the details for `target_host`, `user_id`, `vault_id` etc and finally review & deploy your topology.     
+
+Once you deploy the topology and your topology is valid it will show a success message `Deployed`.
+
+Now below in the `Workflows` section, you will see a dropdown where you can select `run` and click the `Launch` button to finally launch your workflow. You can see all the execution logs from the `Logs` menu.
```
Comment (kinow) [inline Pillar_II/esm/README.md]: :ok_man: perfect, @rohanali1798 ! Just the line breaks, and some suggestions in the text above. Overall looks great! I wish we had docs like this some months ago :slightly_smiling_face: Thank you!!!

**Review by kinow — CHANGES_REQUESTED** [review]

File: Pillar_II/esm/README.md (hunk 1 above)
Comment (rohanali1798) [reply Pillar_II/esm/README.md]: Roger that!
I understand the link breaks. 
What do you mean by "s/a Alien/an Alien"

Comment (kinow) [reply Pillar_II/esm/README.md]: s is commonly used in some utilities to find/replace text. Shorter for suggesting to replace "a Alien" with "an Alien" :slightly_smiling_face:

**Comment by kinow** [comment]
There's also on failing test, which is weird as this is just a doc change. After you push your changes we can check if it's still failing. If so I will have a look. As a final review step I will log in to Alien4Cloud and try your instructions (I think it will work on the first try!).

File: Pillar_II/esm/README.md (hunk 1 above)
Comment (rohanali1798) [reply Pillar_II/esm/README.md]: Thanks for clarifying Bruno. 
Do we need a hard line break using br element or simple Enter would also work?

Comment (kinow) [reply Pillar_II/esm/README.md]: Just a enter should work. I think we have some text that already follows that pattern. You can just try to stay more or less within the same line length/column as the rest of the document.

**Comment by rohanali1798** [comment]
> There's also on failing test, which is weird as this is just a doc change. After you push your changes we can check if it's still failing. If so I will have a look. As a final review step I will log in to Alien4Cloud and try your instructions (I think it will work on the first try!).

Yes, I checked the error, there a was internal server error 500 HTTP while fetching docker image for bats/bats:latest. I think it must be a random network error. Will see again in the next commit.

**Review by kinow — COMMENTED** [review]
Looks great @rohanali1798 ! I'm testing that now on A4C. Fingers crossed it will work (as I haven't used A4C in a long time…)

File: Pillar_II/esm/README.md [hunk 6]
```diff
@@ -348,27 +348,48 @@ may be useful for others with issues:
...
+
+##### Create your application 
+
+Log in to your Alien4Cloud account and create a new application 
+based on an existing topology template. Choose `fesom2 parameterised` 
+as your topology template and proceed creating your application.
+
+##### Configuring topology and running your application
+
+Once you create your application. you will see an option to edit
```
Comment (kinow) [inline Pillar_II/esm/README.md:371]: s/on. y/on, y (comma instead of period)

File: Pillar_II/esm/README.md [hunk 7]
```diff
@@ -348,27 +348,48 @@ may be useful for others with issues:
...
+##### Configuring topology and running your application
+
+Once you create your application. you will see an option to edit 
+your `Environment`. 
+
+Proceed to edit your environment and navigate to `Prepare next deployment`. 
+Here you will see options to modify your `Topology`, `Inputs`, `Location` etc.
+
+Feel free to edit your topology if needed. Since the application we created is 
+based on `fesom2 parameterised` template so it has all the paths and parameters like
```
Comment (kinow) [inline Pillar_II/esm/README.md:378]: s/so / (can drop the so it has, I think)

**Comment by kinow** [comment]
@rohanali1798 great work writing these docs! I followed your steps and got a successful deployment, followed by a successful FESOM2 execution. The workflow is still running on A4C, even though the Slurm job succeeded. Maybe I have to kill it now? Does the same happen to you?

image

What I learned running in Alien4Cloud:

- I get 2 Slurm jobs like you did
- My FESOM2 job finishes in about 2:40 minutes, like before
- The COMPSs job finished in approximately 10 minutes
- The logs of the FESOM2 simulation are in the GPFS shared directory
- But the logs of the COMPSs job are in my home directory
- The UI takes a lot longer to update, and the Logs output is quite messy, and hard to follow; I ended up using squeue and looking at the logs directly on MN4 as I found that more convenient

**Review by kinow — APPROVED** [review]

File: Pillar_II/esm/README.md (hunk 7 above)
Comment (kinow) [reply Pillar_II/esm/README.md:378]: I think I left these comments yesterday, @rohanali1798 . Maybe you can have a look and update these in a next pull request? Merging!
